canonical-websites/www.ubuntu.com
# Remove UbuntuKylin from /desktop and /download sections

## Done

* removed pages from the from the navigation
* removed /desktop/ubuntu-kylin page
* removed /download/ubuntu-kylin page
* removed kylin box from /download
* redirected links to cn.ubuntu.com and cn.ubuntu.com/desktop
* updated the [/download copy doc](https://docs.google.com/document/d/1egsCcmFiz8czeHsvwoB9Jz815Ij5Urtf7Ngrru6Zd4o/edit#heading=h.eps2xiwcgf39)
* updated the [/download desktop copy doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#)
* updated the [/destkop copy doc](https://docs.google.com/document/d/1ZhMkLlkXcimHTIe1taXy24vbbR9pBkZd49H2sJqeQH0/edit#)
* updated the [/desktop/features copy doc](https://docs.google.com/document/d/1nLHJlZ3igDwAAOUDP03wp4iKLxZts8110EA2ZOF5uaU/edit#heading=h.9eff824lzs5m)
* removed all abandoned contextual footers

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at the following pages: 
- [/desktop](http://0.0.0.0:8001/desktop)
- [/desktop/features](http://0.0.0.0:8001/desktop/features)
- [/desktop/ubuntu-kylin](http://0.0.0.0:8001/desktop/ubuntu-kylin) - redirects
- [/download](http://0.0.0.0:8001/download)
- [/download/ubuntu-kylin](http://0.0.0.0:8001/download/ubuntu-kylin) - redirects
